pages/common/gtop.md

```diff
@@ -0,0 +1,19 @@
+# gtop
+
+> System monitoring dashboard for the terminal.
+
+- Show the system stats dashboard:
+
+`gtop`
+
+- Fix question marks or other incorrect characters:
+
+`LANG=en_US.utf8 TERM=xterm-256color gtop`
+
+- Sort by CPU usage:
```

Comment:
I'm not sure it's clear whether this is a command or a keyboard key that you should press. @agnivade, thoughts?

Reply:
Its a keyboard key I would assume. Since the command is similar to `top`

Reply:
Ok then, let's leave it then